src/cargo/core/manifest.rs

```diff
@@ -299,6 +309,7 @@ impl Target {
     pub fn crate_name(&self) -> String { self.name.replace("-", "_") }
     pub fn src_path(&self) -> &Path { &self.src_path }
     pub fn metadata(&self) -> Option<&Metadata> { self.metadata.as_ref() }
+    pub fn features(&self) -> Option<&Vec<String>> { self.features.as_ref() }
```

Comment:
Could this change to returning `Option<&[T]>` instead of `Option<&Vec<T>>`? (generally a little more ergonomic)